index.html

```diff
@@ -8,55 +8,68 @@
     <title>Miguel Cuello's Portfolio</title>
 </head>
 <body>
-
     <header>
       <nav>
+        <div class="left">
         <a class="navbar-logo" href="#">
             My Logo
         </a>
-        <label class="navbar-hamburger" for="toggle">
+        </div>
+
+    <div class="right">   
+            <label class="navbar-hamburger" for="toggle">
             <span class="bar"></span>
             <span class="bar"></span>
             <span class="bar"></span>
-        </label>
-      </nav>
+            </label>  
+        <div id="main_links"> 
+        <a class="main-links">Portfolio</a>
+        <a class="main-links">About</a>
+        <a class="main-links">Contact</a> 
+        </div>
+    </div>
+      </nav> 
     </header>
```

Comment:
- [ ] Kindly consider the following.

- Your design
<img width="441" alt="image" src="https://github.com/migcm06/miguel-portfolio/assets/41163571/46764778-e3c8-4a22-8966-0c4e9246babf">

- Figma design
<img width="407" alt="image" src="https://github.com/migcm06/miguel-portfolio/assets/41163571/267ecf57-2422-481e-a2b7-bf9124290283">

- 1 Kindly increase the text of the items in the first box
- 2 Kindly reduce the gaps between the nav items.